configs/webpack.docs.dev.config.js

```diff
@@ -147,7 +147,8 @@ module.exports = {
         ),
 
         new HtmlWebpackPlugin({
-            template: './docs/index.html'
+            template: './docs/index.html',
```

Comment:
Remove the favicon from index.html as it now gets set here automatically and we have duplicate tags